### PR TITLE
Solidity: naming consistency fixes

### PR DESCRIPTION
**Pull request by sept-en**
The `rainbow-bridge` contracts don't use trailing underscores for the public variables' names, so we need to keep consistency across all connectors.

### CODE REVIEW

**Review by mfornet — APPROVED** [review]
Thanks, I prefer the new convention rather than the old one.

One suggestion, should we add underscore at the end per [Syte Guide](https://docs.soliditylang.org/en/v0.8.10/style-guide.html?#avoiding-naming-collisions). Not blocking.

**Comment by sept-en** [comment]
@mfornet we don't use those trailing underscores in most other contracts so the initial goal is to get consistency. And also the drawback I think is that it's not that convenient using it from API/Etherscan.
However, I think that we may consider changing everything to underscores too, WDYT?

File: eth-custodian/scripts/eth_get_prover_address.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const hre = require('hardhat');
 
 const erc20ConnectorAbi = require('./json/erc20-connector.json');
-const erc20ConnectorAddress = '0xa5289b6d5dcc13e48f2cc6382256e51589849f86';
+const erc20ConnectorAddress = '0xc115851ca60aed2ccc6ee3d5343f590834e4a3ab';
```
Comment (karim-en) [inline eth-custodian/scripts/eth_get_prover_address.js:4]: Looks like this line was edited by mistake, please check

**Review by karim-en — APPROVED** [review]

File: eth-custodian/scripts/eth_get_prover_address.js (hunk 1 above)
Comment (karim-en) [inline eth-custodian/scripts/eth_get_prover_address.js:4]: Looks like this line was edited by mistake, please check

**Review by karim-en — CHANGES_REQUESTED** [review]

File: eth-custodian/scripts/eth_get_prover_address.js (hunk 1 above)
Comment (sept-en) [inline eth-custodian/scripts/eth_get_prover_address.js:4]: Yes, that's a mistake.
```suggestion
const erc20ConnectorAddress = '0xa5289b6d5dcc13e48f2cc6382256e51589849f86';
```

**Review by karim-en — APPROVED** [review]

**Pull request merged**